public/index.html

```diff
@@ -12,6 +12,7 @@
   <link rel="apple-touch-icon" sizes="180x180" href="%PUBLIC_URL%/favicon/apple-touch-icon.png">
   <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicon/favicon-32x32.png">
   <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicon/favicon-16x16.png">
+  <link href="https://fonts.googleapis.com/css?family=Overpass" rel="stylesheet">
```

Comment:
Are you sure we have to place it here? Maybe we can put it somewhere in polymath-ui?

Reply:
okay, I will make a PR in polymath ui for this.

so it is now removed from whitelist

Reply:
https://github.com/PolymathNetwork/polymath-ui/blob/75430df5905ff282ab96dcdec548bb6d9f152427/src/style.scss#L2 it already exists in polymath ui here